tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
```

Comment:
Can you change this to 

```json
"lib": ["es2020"],
"module": "commonjs",
"target": "es2020",
```

to match what Node.js 14 supports? (https://www.npmjs.com/package/@tsconfig/node14)

Reply:
We use modules instead of commonJs so changed to be es2020 to all 3 of them.